lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -218,6 +218,9 @@ module InternalConstants
     # Whether to attempt to obtain metadata from the local metadata service.
     # It is safe to specify 'true' even on platforms with no metadata service.
     config_param :use_metadata_service, :bool, :default => true
+    # Whether to set the AWS location to the availability zone instead of the
+    # region.
+    config_param :use_aws_availability_zone, :bool, :default => true
```

Comment:
Should we also give recommendation/clear preference in the comment that `region` is the right thing while `availability zone` is some legacy residue that we are looking forward to deprecate?

Reply:
Also, we meant to leave `default` as `true` for the moment, I assume?